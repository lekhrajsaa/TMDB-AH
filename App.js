import React from 'react';
import {StyleSheet} from 'react-native';
import {NavigationContainer} from '@react-navigation/native';
import StackNavigation from './src/navigation/StackNavigation';
import {store} from './src/store/store';
import {Provider} from 'react-redux';

const App = props => {
  return (
    <Provider store={store}>
      <NavigationContainer>
        <StackNavigation />
      </NavigationContainer>
    </Provider>
  );
};

const styles = StyleSheet.create({});

export default App;
