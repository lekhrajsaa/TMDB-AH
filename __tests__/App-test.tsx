import React from 'react';
import renderer from 'react-test-renderer';
import {fireEvent, render} from '@testing-library/react-native';
import App from '../App';

test('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();

  // const {getByTestId} = render(<HomeScreen navigation={undefined} />);
  // const button = getByTestId('test');
  // const text = getByTestId('text');

  // fireEvent.changeText(text, 'Showing');
  // fireEvent.press(button);
});
