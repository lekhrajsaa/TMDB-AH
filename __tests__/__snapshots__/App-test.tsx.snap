// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  nestedScrollEnabled={true}
  style={
    {
      "flex": 1,
      "paddingTop": 20,
    }
  }
>
  <View
    style={
      {
        "backgroundColor": "#F5F5F5",
        "height": 1334,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": 187.5,
      }
    }
  />
  <View
    style={
      {
        "backgroundColor": "#ffffff",
        "height": 1334,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "width": 562.5,
      }
    }
  />
  <View
    style={
      {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginHorizontal": 16,
        "padding": 8,
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="24"
        bbWidth="24"
        fill="none"
        focusable={false}
        height="24"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 24,
              "width": 24,
            },
          ]
        }
        vbHeight={24}
        vbWidth={24}
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M11 5.75C11 5.33579 11.3358 5 11.75 5H19.25C19.6642 5 20 5.33579 20 5.75C20 6.16421 19.6642 6.5 19.25 6.5H11.75C11.3358 6.5 11 6.16421 11 5.75ZM4 11.75C4 11.3358 4.33579 11 4.75 11H19.25C19.6642 11 20 11.3358 20 11.75C20 12.1642 19.6642 12.5 19.25 12.5H4.75C4.33579 12.5 4 12.1642 4 11.75ZM4.75 17C4.33579 17 4 17.3358 4 17.75C4 18.1642 4.33579 18.5 4.75 18.5H12.25C12.6642 18.5 13 18.1642 13 17.75C13 17.3358 12.6642 17 12.25 17H4.75Z"
            fill={
              {
                "payload": 4279307847,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
    <Text
      style={
        {
          "alignSelf": "center",
          "color": "#110E47",
          "fontFamily": "Merriweather-Regular",
          "fontSize": 16,
          "fontWeight": "900",
        }
      }
      testID="text"
    >
      FilmKu
    </Text>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="24"
        bbWidth="24"
        fill="none"
        focusable={false}
        height="24"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 24,
              "width": 24,
            },
          ]
        }
        vbHeight={24}
        vbWidth={24}
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M14.9396 18.4767H18.5999C19.372 18.4767 20 17.8164 20 17.0047C20 16.574 19.8216 16.1661 19.5103 15.8869C19.4911 15.8692 19.4703 15.8532 19.4488 15.8381C18.2728 14.7597 17.6 13.2088 17.6 11.571V10.6543C17.6 10.3061 17.3312 10.0235 17 10.0235C16.6688 10.0235 16.4 10.3061 16.4 10.6543V11.571C16.4 13.6032 17.2464 15.5251 18.7224 16.8456C18.7254 16.8482 18.7282 16.8506 18.7311 16.8531C18.7397 16.8604 18.7478 16.8674 18.7568 16.8743C18.7792 16.9029 18.8 16.9458 18.8 17.0047C18.8 17.119 18.7087 17.215 18.5999 17.215H5.39995C5.29126 17.215 5.2 17.119 5.2 17.0047C5.2 16.9467 5.2208 16.9029 5.24233 16.8743C5.25296 16.8667 5.2619 16.859 5.27145 16.8509C5.27347 16.8492 5.27553 16.8474 5.27763 16.8456C6.75273 15.5261 7.59999 13.6032 7.59999 11.571V9.22428C7.59999 6.67319 9.57445 4.59817 11.9999 4.59817C12.0119 4.59817 12.0241 4.59805 12.0364 4.59792C12.0895 4.5974 12.1456 4.59684 12.1975 4.60571C12.5183 4.66301 12.8336 4.43075 12.8879 4.08761C12.9424 3.74447 12.7215 3.41888 12.3944 3.36175C12.2664 3.33895 12.128 3.33649 11.9999 3.33649C8.91278 3.33649 6.39999 5.97767 6.39999 9.22428V11.571C6.39999 13.2078 5.72792 14.758 4.55356 15.8355C4.53115 15.8507 4.50962 15.8675 4.48955 15.886C4.17842 16.1661 4 16.574 4 17.0047C4 17.8164 4.62798 18.4767 5.39995 18.4767H9.06023C9.33905 19.9148 10.5517 21 11.9999 21C13.4483 21 14.6608 19.9148 14.9396 18.4767ZM13.6971 18.4767H10.3028C10.5505 19.2111 11.2181 19.7383 11.9999 19.7383C12.7818 19.7383 13.4494 19.2111 13.6971 18.4767Z"
            fill={
              {
                "payload": 4279307847,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
          <RNSVGPath
            d="M17.0001 9C15.3458 9 14 7.65428 14 6.00005C14 4.34583 15.3458 3 17.0001 3C18.6543 3 20 4.34583 20 6.00005C20 7.65428 18.6543 9 17.0001 9Z"
            fill={
              {
                "payload": 4294913582,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
  </View>
  <View>
    <View
      style={
        {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginHorizontal": 16,
          "marginTop": 20,
          "padding": 8,
        }
      }
    >
      <Text
        style={
          {
            "alignSelf": "center",
            "color": "#110E47",
            "fontFamily": "Merriweather-Regular",
            "fontSize": 16,
            "fontWeight": "900",
          }
        }
      >
        Now Showing
      </Text>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "",
            "borderColor": "#E5E4EA",
            "borderRadius": 12,
            "borderWidth": 1,
            "justifyContent": "center",
            "marginEnd": 0,
            "opacity": 1,
            "paddingHorizontal": 8,
            "paddingVertical": 4,
          }
        }
      >
        <Text
          style={
            {
              "color": "#AAA9B1",
              "fontFamily": "Mulish-Regular",
              "fontSize": 10,
              "fontWeight": "400",
              "lineHeight": 12,
            }
          }
        >
          See more
        </Text>
      </View>
    </View>
    <RCTScrollView
      contentContainerStyle={
        {
          "paddingEnd": 12,
        }
      }
      data={[]}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={true}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      showsHorizontalScrollIndicator={false}
      stickyHeaderIndices={[]}
      style={
        {
          "flexGrow": 0,
          "marginStart": 24,
          "marginTop": 8,
        }
      }
      viewabilityConfigCallbackPairs={[]}
    >
      <View />
    </RCTScrollView>
  </View>
  <View
    style={
      {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginHorizontal": 16,
        "marginTop": 20,
        "padding": 8,
      }
    }
  >
    <Text
      style={
        {
          "alignSelf": "center",
          "color": "#110E47",
          "fontFamily": "Merriweather-Regular",
          "fontSize": 16,
          "fontWeight": "900",
        }
      }
    >
      Popular
    </Text>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "",
          "borderColor": "#E5E4EA",
          "borderRadius": 12,
          "borderWidth": 1,
          "justifyContent": "center",
          "marginEnd": 0,
          "opacity": 1,
          "paddingHorizontal": 8,
          "paddingVertical": 4,
        }
      }
    >
      <Text
        style={
          {
            "color": "#AAA9B1",
            "fontFamily": "Mulish-Regular",
            "fontSize": 10,
            "fontWeight": "400",
            "lineHeight": 12,
          }
        }
      >
        See more
      </Text>
    </View>
  </View>
  <RCTScrollView
    contentContainerStyle={
      {
        "paddingBottom": 0,
      }
    }
    data={[]}
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsVerticalScrollIndicator={false}
    stickyHeaderIndices={[]}
    style={
      {
        "flexGrow": 0,
        "marginStart": 24,
        "marginTop": 8,
      }
    }
    viewabilityConfigCallbackPairs={[]}
  >
    <View />
  </RCTScrollView>
  <View
    onLayout={[Function]}
    style={
      {
        "backgroundColor": "#ffffff",
        "borderTopColor": "#E5E4EA",
        "borderTopWidth": 1,
        "bottom": 0,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingHorizontal": 40,
        "paddingVertical": 8,
        "position": "absolute",
        "width": "100%",
        "zIndex": 1,
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="35"
        bbWidth="35"
        fill="none"
        focusable={false}
        height="35"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 35,
              "width": 35,
            },
          ]
        }
        vbHeight={35}
        vbWidth={35}
        width="35"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGGroup
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
          >
            <RNSVGPath
              d="M28.12 12.56C28.12 6.7373 23.3827 2 17.56 2C11.7373 2 7 6.7373 7 12.56C7 18.3827 11.7373 23.12 17.56 23.12H26.8V21.8H22.661C25.9122 19.9979 28.12 16.5331 28.12 12.56Z"
              fill={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                  "stroke",
                ]
              }
              stroke={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
            />
            <RNSVGPath
              d="M8.32001 12.56C8.32001 7.46502 12.465 3.32001 17.56 3.32001C22.6551 3.32001 26.8 7.46498 26.8 12.56C26.8 17.655 22.655 21.8 17.56 21.8C12.465 21.8 8.32001 17.655 8.32001 12.56Z"
              fill={
                {
                  "payload": 4292269821,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                  "stroke",
                ]
              }
              stroke={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
            />
            <RNSVGPath
              d="M19.672 8.336C19.672 9.50061 18.7246 10.448 17.56 10.448C16.3953 10.448 15.448 9.50061 15.448 8.336C15.448 7.17139 16.3954 6.224 17.56 6.224C18.7246 6.224 19.672 7.17139 19.672 8.336ZM18.264 8.33602C18.264 7.94759 17.9484 7.63202 17.56 7.63202C17.1715 7.63202 16.856 7.94759 16.856 8.33602C16.856 8.72445 17.1716 9.04002 17.56 9.04002C17.9484 9.04002 18.264 8.72445 18.264 8.33602Z"
              fill={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            />
            <RNSVGPath
              d="M13.9891 8.86155C15.096 9.22182 15.7037 10.416 15.3449 11.5228C15.0547 12.4145 14.2242 12.9824 13.334 12.9824C13.1181 12.9824 12.8994 12.9494 12.6836 12.8793C11.576 12.519 10.9683 11.3256 11.3271 10.218C11.6874 9.11044 12.8823 8.50269 13.9885 8.86155H13.9891ZM13.3367 10.1664C13.0397 10.1664 12.7626 10.3555 12.6664 10.6532C12.5468 11.0216 12.7495 11.4204 13.1187 11.5401C13.4845 11.6597 13.886 11.4576 14.0056 11.0877C14.1252 10.7192 13.9224 10.3211 13.5539 10.2008C13.4818 10.1774 13.4089 10.1664 13.3367 10.1664Z"
              fill={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            />
            <RNSVGPath
              d="M13.3178 14.6964C14.0026 13.7557 15.3262 13.5453 16.2682 14.2288C17.2095 14.9136 17.4193 16.2373 16.7358 17.1793C16.4037 17.6366 15.9142 17.9364 15.3565 18.0244C15.2451 18.0423 15.1331 18.0512 15.0217 18.0512C14.5795 18.0512 14.1498 17.9123 13.7847 17.6469C13.3281 17.3155 13.0283 16.8259 12.9403 16.2683C12.8516 15.7113 12.9857 15.153 13.3178 14.6964ZM15.1364 16.634C15.3221 16.6045 15.4857 16.5041 15.5964 16.3521C15.824 16.0379 15.7546 15.5964 15.4404 15.3682C15.3159 15.2782 15.1708 15.2341 15.0271 15.2341C14.8092 15.2341 14.5946 15.3352 14.4564 15.5243C14.3464 15.6762 14.3017 15.8626 14.3306 16.0482C14.3601 16.2339 14.4605 16.3975 14.6125 16.5082C14.7644 16.6182 14.9522 16.6595 15.1364 16.634Z"
              fill={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            />
            <RNSVGPath
              d="M18.3842 17.1794C17.7007 16.2376 17.9104 14.914 18.8517 14.23C19.7938 13.5452 21.1187 13.7549 21.8022 14.6968C22.1343 15.1533 22.2684 15.7116 22.1797 16.2685C22.0917 16.8261 21.7919 17.3156 21.3353 17.647C20.9709 17.9123 20.5404 18.0512 20.0983 18.0512C19.9869 18.0512 19.8749 18.0423 19.7635 18.0244C19.2058 17.9364 18.7163 17.6366 18.3849 17.1801L18.3842 17.1794ZM19.9835 16.6342C20.1692 16.661 20.3548 16.6191 20.5075 16.5084C20.6595 16.3977 20.7598 16.2341 20.7894 16.0484C20.8183 15.8628 20.7736 15.6765 20.6636 15.5246C20.4353 15.2104 19.9918 15.1403 19.6796 15.3692C19.3661 15.5968 19.2959 16.0382 19.5235 16.3524C19.6342 16.5043 19.7979 16.6047 19.9835 16.6342Z"
              fill={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            />
            <RNSVGPath
              d="M19.7751 11.5236C19.4163 10.416 20.024 9.22183 21.1309 8.86156C22.2378 8.50269 23.4326 9.11045 23.7929 10.2173C24.1517 11.3255 23.544 12.519 22.4371 12.8793H22.4365C22.2206 12.9494 22.0019 12.9824 21.7861 12.9824C20.8958 12.9824 20.0653 12.4145 19.7751 11.5236ZM21.1144 11.0883C21.234 11.4575 21.6314 11.6603 22.0019 11.54C22.3704 11.4197 22.5732 11.0216 22.4536 10.6524C22.3574 10.3554 22.0803 10.1664 21.7833 10.1664C21.7111 10.1664 21.6382 10.1774 21.5661 10.2008C21.1975 10.3211 20.9947 10.7191 21.1144 11.0883Z"
              fill={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            />
            <RNSVGPath
              d="M18.456 11.6639C18.9509 12.1588 18.9509 12.9611 18.456 13.456C17.9612 13.9509 17.1588 13.9509 16.6639 13.456C16.1691 12.9611 16.1691 12.1588 16.6639 11.6639C17.1588 11.1691 17.9612 11.1691 18.456 11.6639Z"
              fill={
                {
                  "payload": 4279307847,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            />
          </RNSVGGroup>
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="25px"
        bbWidth="25px"
        focusable={false}
        height="25px"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 25,
              "width": 25,
            },
          ]
        }
        vbHeight={50}
        vbWidth={50}
        width="25px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={
            {
              "payload": 4278190080,
              "type": 0,
            }
          }
        >
          <RNSVGPath
            d="M 21 3 C 11.621094 3 4 10.621094 4 20 C 4 29.378906 11.621094 37 21 37 C 24.710938 37 28.140625 35.804688 30.9375 33.78125 L 44.09375 46.90625 L 46.90625 44.09375 L 33.90625 31.0625 C 36.460938 28.085938 38 24.222656 38 20 C 38 10.621094 30.378906 3 21 3 Z M 21 5 C 29.296875 5 36 11.703125 36 20 C 36 28.296875 29.296875 35 21 35 C 12.703125 35 6 28.296875 6 20 C 6 11.703125 12.703125 5 21 5 Z"
            fill={
              {
                "payload": 4290559181,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="24"
        bbWidth="24"
        fill="none"
        focusable={false}
        height="24"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 24,
              "width": 24,
            },
          ]
        }
        vbHeight={24}
        vbWidth={24}
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGRect
            fill={
              {
                "payload": 4292401368,
                "type": 0,
              }
            }
            height="24"
            opacity={0.01}
            propList={
              [
                "fill",
              ]
            }
            width="24"
            x="0"
            y="0"
          />
          <RNSVGPath
            d="M19.1002 2.11908C19.375 2.23363 19.5939 2.41419 19.7564 2.66111C19.9187 2.90789 20 3.18096 20 3.48065V20.5195C20 20.8192 19.9188 21.0923 19.7564 21.3389C19.5938 21.586 19.3749 21.7667 19.1002 21.8811C18.9418 21.9517 18.7584 21.9867 18.55 21.9867C18.1502 21.9867 17.8043 21.8456 17.5126 21.5641L12 15.9589L6.48747 21.564C6.18739 21.8545 5.84173 22 5.44992 22C5.25829 22 5.07489 21.9604 4.90002 21.8812C4.62507 21.7668 4.40625 21.586 4.24378 21.3393C4.08135 21.0923 4 20.8193 4 20.5195V3.48047C4 3.18077 4.08135 2.9077 4.24378 2.66092C4.40625 2.41415 4.62507 2.23335 4.90002 2.1189C5.07506 2.03968 5.25829 2 5.44992 2H18.5501V2.00019C18.7418 2.00019 18.9252 2.03986 19.1002 2.11908ZM13.1126 14.7427L18.4001 20.1095V3.6922H5.60016V20.1095L10.8876 14.7427L12.0001 13.6193L13.1126 14.7427Z"
            fill={
              {
                "payload": 4290559181,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
  </View>
</View>
`;
